packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -511,7 +510,7 @@ $Calendar-dayPicker-margin: 10px;
 
 .wrap span:focus,
 .wrap div:focus {
-  outline: 1px solid $ms-color-themePrimary;
+  outline: $ms-color-themePrimary;
```

Comment:
Tried to change it to use the mixin but it ends up breaking the layout of the picker because the mixin is using relative positioning, so for now my preference would be to leave it as-is.